utils/pre-commit

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
I had to change this by `#!/bin/bash` in order to make this work. With `sh` I get the error
```
Checking format before committing
.git/hooks/pre-commit: 21: Syntax error: "(" unexpected
```

Reply:
Could you please make a PR with this change?